engine/engine.go

```diff
@@ -52,19 +53,22 @@ func (e *Engine) Setup(ctx context.Context, pipelineConfig *spec.PipelineConfig)
 	}
 	// create volumes
 	for _, vol := range pipelineConfig.Volumes {
-		if vol != nil && vol.HostPath != nil {
-			path := vol.HostPath.Path
-			vol.HostPath.Path = pathConverter(path)
+		if vol == nil || vol.HostPath == nil {
```

Comment:
Had to make this change due to failed linter

https://harness.drone.io/harness/lite-engine/930/1/3